### PR TITLE
Password encoding

### PR DESCRIPTION
**Pull request by MikeMoudry**

### CODE REVIEW

File: Backend-API/src/main/java/com/avangers/backendapi/service/UserService.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+@Service
+public class UserService {
+
+    @Autowired
+    private PasswordEncoder passwordEncoder;
+
+    public void registerUser(String rawPassword) {
+        String encodedPassword = passwordEncoder.encode(rawPassword);
+    }
```
Comment (Normelik) [inline Backend-API/src/main/java/com/avangers/backendapi/service/UserService.java:15]: maybe use the @RequiredArgsConstructor?

Comment (MikeMoudry) [reply Backend-API/src/main/java/com/avangers/backendapi/service/UserService.java:15]: great idea

**Review by markkovari — APPROVED** [review]
image

**Review by ReneVajda1 — APPROVED** [review]
approved

**Pull request merged**